Resources/Prototypes/_RMC14/Entities/Clothing/visors.yml

```diff
@@ -151,3 +146,97 @@
     toggledSprite:
       sprite: _RMC14/Objects/Clothing/Head/Helmets/huds.rsi
       state: hud_sight_left
+
+- type: entity
+  parent: RMCVisorBase
+  id: RMCVisorPilotBlack
+  name: MK30 black flight visor
+  description: A standard issue snap-on visor used by UNMC dropship pilots. Polarized to reduce glare and protect the eyes during atmospheric re-entry and orbital deployment.
+  components:
+  - type: Sprite
+    state: po_visor
+  - type: Visor
+    toggledSprite:
+      sprite: _RMC14/Objects/Clothing/Head/Helmets/huds.rsi
+      state: po_visor_black
+    onIcon:
+      sprite: _RMC14/Objects/Clothing/helmet_visors.rsi
+      state: po_visor_down
+  - type: ComponentToggler
+    components:
+    - type: FlashImmunity
```

Comment:
Unless there's specifically a mechanic that flashes pilots, this should be removed

No reason to add an item outside of command roles that prevents flashes by mps